packages/queued-request-controller/package.json

```diff
@@ -32,16 +32,16 @@
   },
   "dependencies": {
     "@metamask/base-controller": "^4.1.1",
-    "@metamask/controller-utils": "^8.0.3",
-    "@metamask/json-rpc-engine": "^7.3.2",
+    "@metamask/controller-utils": "^8.0.4",
+    "@metamask/json-rpc-engine": "^7.3.3",
     "@metamask/rpc-errors": "^6.2.1",
     "@metamask/swappable-obj-proxy": "^2.2.0",
     "@metamask/utils": "^8.3.0"
   },
   "devDependencies": {
-    "@metamask/approval-controller": "^5.1.2",
+    "@metamask/approval-controller": "^5.1.3",
     "@metamask/auto-changelog": "^3.4.4",
-    "@metamask/network-controller": "^17.2.0",
+    "@metamask/network-controller": "^17.2.1",
```

Comment:
Are the peer dependencies here not updated due to https://github.com/MetaMask/core/pull/3881?